syntax = "proto3";

message InputMessage {
    string msg = 1;
    string dest = 2;
    string origin = 3;
}

message ReponseMessage{
    string action = 1;
    InputMessage msg = 2;
}

message Empty{

}

service DataTransfer {
    rpc sendMessage(InputMessage) returns (Empty);
    rpc recvMessage(Empty) returns (ReponseMessage);
}

//python -m grpc.tools.protoc -I. --python_out=. --grpc_python_out=. sample.proto

